modules/contenta_graphql/composer.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
I don't think our module really needs a composer.json. Is that actually read by composer at all?